pkgs/by-name/ln/lnav/package.nix

```diff
@@ -48,10 +54,18 @@ stdenv.mkDerivation rec {
     readline
     sqlite
     curl
-    gpm
     libarchive
+  ] ++ lib.optionals (!stdenv.isDarwin) [
+    gpm
   ];
 
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    src = "${src}/src/third-party/prqlc-c";
```

Comment:
Does it need this even if the `cargoRoot` is specified? If so, that should probably be fixed.

Reply:
You are right!

Reply:
Looking at @symphorien's build error, this likely caused it and you merely didn't notice because the FOD was cached.

Reply:
I expected the change to invalidate the cache. Why wasn't that the case?

Reply:
Because the FOD hash was still the same. Once a FOD with the same hash and name exists in the Nix store, Nix will not attempt to realise it again.